### DISCUSSION TITLE
[FR] <Add a option to the script to restore from update>

### DISCUSSION THREAD

**Discussion by modhash** [pivpn/pivpn#1383] [Feature requests]
**Is your feature request related to a problem? Please describe.**
Its very annoying to restore using  https://github.com/pivpn/pivpn/wiki/WireGuard#how-can-i-migrate-my-configs-to-another-pivpn-instance It would be way easier if it can just be included in the script.

**Describe the solution you'd like**
Include a option at install time and a command to restore from update in the regular command.

**Describe alternatives you've considered**
I did it manually but its a real pain.

**Comment by ghost** [comment]
If this is done, can a version number be stored in the saved configuration, so that the restore can check compatibility.

**Comment by coolapso** [comment]
going to investigate a way to do this ... 

however I am not too fancy about adding this one! 
See a lot of ways where this can go wrong.

**Comment by modhash** [comment]
why we could make it a single config file with all the key pairs and names inside
